src/main/resources/hudson/plugins/ec2/SlaveTemplate/config.jelly

```diff
@@ -223,6 +223,15 @@ THE SOFTWARE.
 
     <f:descriptorList title="${%Node Properties}" field="nodeProperties" descriptors="${descriptor.nodePropertyDescriptors}" />
 
+    <f:entry title="${%Enable Metadata HTTP Endpoint}" field="metadataEndpointEnabled" >
+      <f:checkbox default="true"/>
+    </f:entry>
+    <f:entry title="${%Metadata Require HTTP Tokens}" field="metadataTokensRequired">
+      <f:checkbox default="false"/>
+    </f:entry>
+    <f:entry title="${%Metadata Put Response Hop Limit}" field="metadataHopsLimit">
```

Comment:
Are there any restrictions to the value in this field?

Reply:
1-64 per Amazon.  ae7ba08